Cargo.toml

```diff
@@ -8,11 +8,11 @@ license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
 
 [dependencies]
-byteorder = "~1.2.1"
+byteorder = "~1.3.1"
 log = "0.4"
-kvm-bindings = "~0"
-kvm-ioctls = "~0"
+kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch" }
```

Comment:
Why there's special dependency on the kvm-bindings/kvm-ioclts?

Reply:
Cloud Hypervisor have dependencies for the KVM serialization work,  kvm-bindings/kvm-ioclts related patches will submit to upstream, after that we can use the same base, else we will have building error due to different versions.

Reply:
@arronwy fwiw, we don't need the cloud-hypervisor version of kvm-bindings to submit this upstream.

Reply:
I found if we use the upstream kvm-binding, cloud-hypervisor will failed to build with following error:
```
error[E0308]: mismatched types
   --> vfio-ioctls/src/vfio_device.rs:200:30
    |
200 |  .set_device_attr(&dev_attr)
    |   ^^^^^^^^^ expected struct `kvm_bindings::x86::bindings_v4_20_0::kvm_device_attr`, found struct `kvm_bindings::kvm_device_attr`
    |
    = note: expected reference `&kvm_bindings::x86::bindings_v4_20_0::kvm_device_attr`
               found reference `&kvm_bindings::kvm_device_attr`
    = note: perhaps two different versions of crate `kvm_bindings` are being used?
```